### PR TITLE
Add a link to the cherry-pick-pr script from gardener

### PR DESCRIPTION
**Pull request by vpnachev**
**How to categorize this PR?**

/area dev-productivity
/kind enhancement
/platform aws

**What this PR does / why we need it**:
This allows to use the same approach for creating automatic cherry-pick PRs like g/g. 
Note, this was possible even today, but the script has to be invoked as `./vendor/github.com/gardener/gardener/hack/cherry-pick-pull.sh` instead of `./hack/cherry-pick-pull.sh`

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other operator

```

### CODE REVIEW

File: hack/cherry-pick-pull.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+/home/I311840/go/src/github.com/gardener/gardener-extension-provider-aws/vendor/github.com/gardener/gardener/hack/cherry-pick-pull.sh
```
Comment (prashanth26) [inline hack/cherry-pick-pull.sh:1]: Is this change required? :o Mistake?

**Review by prashanth26 — COMMENTED** [review]
/lgtm

File: hack/cherry-pick-pull.sh (hunk 1 above)
Comment (vpnachev) [reply hack/cherry-pick-pull.sh]: Yes, but it should be relative path. It is fixed now.

**Review by prashanth26 — APPROVED** [review]
/lgtm

**Review by rfranzke — APPROVED** [review]
